src/jobs/membership.ts

```diff
@@ -0,0 +1,51 @@
+import cron from 'node-cron';
+import User from '../models/user';
+import { MEMBERSHIP_STATUS } from '../types/user';
+import {
+  GoogleSpreadsheet,
+  WorksheetBasicProperties,
+} from 'google-spreadsheet';
+import GoogleCreds from '../config/acs-web-5aaaf-768e669e2d39.json';
+
+const getTerm = () => {
+  const date = new Date();
+  const month = date.getMonth();
+  if (month === 1) {
+    return 'Fall' + (date.getFullYear() - 1);
+  } else if (month === 5) {
+    return 'Winter' + date.getFullYear();
+  } else if (month === 9) {
+    return 'Spring' + date.getFullYear();
+  }
+};
+
+const doc = new GoogleSpreadsheet(
+  '1Kn3Fc1vmi2dd9I645BxBQZMXXNvIJi1zEGnBiHuHDtk',
+);
+
+cron.schedule('0 0 1 JAN,MAY,SEP *', async function () {
+  try {
+    await doc.useServiceAccountAuth(GoogleCreds);
+    await doc.loadInfo();
+    const newSheet = await doc.addSheet({
+      title: getTerm(),
+    } as WorksheetBasicProperties);
```

Comment:
How come we need a cast here? btw the PR to definitely typed is approved and I think just waiting for another review by the maintainer of `google-spreadsheet`.

Reply:
Good news! https://github.com/theoephraim/node-google-spreadsheet/pull/287#issuecomment-648273964
I'll make a PR upgrading the package and see if that resolves this